react/components/FilterSidebar.js

```diff
@@ -200,6 +209,20 @@ const FilterSidebar = ({
               <FormattedMessage id="store/search-result.filter-button.apply" />
             </Button>
           </div>
+          {totalProductsOnMobile === 'show' && recordsFiltered && (
+            <div
+              className={`${handles.filterTotalProducts} w-100 flex flex-grow-1 items-center justify-center pre t-action--small`}
```

Comment:
It's not an action, so I suggest to change to something more semantic
https://styleguide.vtex.com/#/Styles?id=section-typography

```suggestion
              className={`${handles.filterTotalProducts} w-100 flex flex-grow-1 items-center justify-center pre t-small`}
```